doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -0,0 +1,495 @@
+    DPU2                8K-DPU400-2A    0.10            FLM274801EY     400G DPU 
+    DPU3                8K-DPU400-2A    0.10            FLM27500371     400G DPU
+
+Power Supplies                                                                
+    psutray                                                                                                                                                             
+        PSU0            PSUXKW-ACPI     0.0             POG2427K01K     AC Power Module with Port-side Air Intake                                                 
+        PSU1            PSUXKW-ACPI     0.0             POG2427K00Y     AC Power Module with Port-side Air Intake                                                 
+
+Cooling Devices
+    fantray0            FAN-2RU-PI-V3   N/A             N/A             8000 Series 2RU Fan 
+    fantray1            FAN-2RU-PI-V3   N/A             N/A             8000 Series 2RU Fan 
+    fantray2            FAN-2RU-PI-V3   N/A             N/A             8000 Series 2RU Fan 
+    fantray3            FAN-2RU-PI-V3   N/A             N/A             8000 Series 2RU Fan
+
+FPDs
+    RP0/info.0                          0.5.6-253      
+
+```
+
+show platform temperature - shows the DPU temperature on the switch
+```
+root@sonic:~#show platform temperature
+
+         Sensor    Temperature    High TH    Low TH    Crit High TH    Crit Low TH    Warning          Timestamp
+---------------  -------------  ---------  --------  --------------  -------------  ---------  -----------------
+        DPU_0_T         37.438      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+        DPU_1_T         37.563      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+        DPU_2_T           38.5      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+        DPU_3_T         38.813      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+     FAN_Sensor         23.201      100.0      -5.0           102.0          -10.0      False  20230728 06:39:18
+ MB_PORT_Sensor         21.813       97.0      -5.0           102.0          -10.0      False  20230728 06:39:18
+MB_TMP421_Local          26.25      135.0      -5.0           140.0          -10.0      False  20230728 06:39:18
+       SSD_Temp           40.0       80.0      -5.0            83.0          -10.0      False  20230728 06:39:18
+   X86_CORE_0_T           37.0      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+   X86_CORE_1_T           37.0      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+   X86_PKG_TEMP           41.0      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+```
+
+show platform fan - shows the fan speed and status on the switch
+```
+root@sonic:~#show platform fan
```

Comment:
@rameshraghupathy Please indicate preferably at the top right corner in red ink where this CLI is executed: DPU or NPU

Reply:
Done